.changeset/tough-icons-push.md

```diff
@@ -0,0 +1,5 @@
+---
+'@protocolink/api': patch
+---
+
+add IOLEND and Wagmi test and example
```

Comment:
```suggestion
add IOLEND test and example
```
or maybe we can just remove `.changeset/odd-timers-swim.md`

Reply:
fixed. thanks.